service/src/test/java/uk/gov/hmcts/reform/fpl/service/CaseDataExtractionServiceTest.java

```diff
@@ -43,29 +48,69 @@
         hearingBookingService, hmctsCourtLookupConfiguration);
 
     @Test
-    void shouldConcatenateAllChildrenNames() {
-        CaseData caseData = CaseData.builder()
-            .caseLocalAuthority("example")
-            .familyManCaseNumber("123")
+    void shouldApplySentenceFormattingToMultipleChildNames() {
```

Comment:
hmm I prefer should... when.. tests

shouldApplySentenceFormattingWhenMultplieChildrenNames etc :)